publication.qmd

```diff
@@ -0,0 +1,72 @@
+---
+title: "The Journal of Visualization and Interaction"
+bibliography: biblio.bib
+subtitle: A [diamond open-access](https://en.wikipedia.org/wiki/Open_access#Diamond/platinum_OA) journal for the [InfoVis](http://ieeevis.org/) and [HCI](https://chi.acm.org/) communities.
+author: 
+  - "[Lonni Besançon](https://lonnibesancon.me/), Linköping University"
+  - "[Florian Echtler](https://floe.butterbrot.org/), Aalborg University"
+  - "[Matthew Kay](https://www.mjskay.com/), Northwestern University"
+  - "Chat Wacharamanotham, Swansea University"
+format:
+  pdf:
+    link-citations: true
+---
+
+# What is JoVI?
```

Comment:
Is there a way to provide section number in the generated PDF? The numbered sections are easier to refer to during the reviewing process.

Reply:
done